beatrix/src/test/java/org/killbill/billing/beatrix/util/InvoiceChecker.java

```diff
@@ -170,6 +180,26 @@ public void checkChargedThroughDate(final UUID entitlementId, final LocalDate ex
         }
     }
 
+
+    public void checkTrackingIds(final Invoice invoice, final Set<String> expectedTrackingIds, final InternalCallContext internalCallContext) {
+        final InvoiceTrackingSqlDao dao = dbi.onDemand(InvoiceTrackingSqlDao.class);
```

Comment:
Might be worth creating a bug already for 0.22 to introduce a real API and fix tests.